lib/imagemounter/index.js

```diff
@@ -0,0 +1,114 @@
+import { BaseServicePlist } from '../base-service';
+import { fs } from '@appium/support';
+import B from 'bluebird';
+import log from '../logger';
+const MOBILE_IMAGE_MOUNTER_SERVICE_NAME = 'com.apple.mobile.mobile_image_mounter';
+const { lstat, readFile, createReadStream } = fs;
+const FILE_TYPE_IMAGE = 'image';
+const FILE_TYPE_SIGNATURE = 'signature';
+function _checkReturnError(ret) {
```

Comment:
I don't see any need to use underscore prefix for function names. They are anyway private to the module unless you export them explicitly. It is not python %)

Reply:
how about checkIfError